File: teoricos/tt2/processoAEDSIIVerde/Matriz Java/Matriz.java
class Matriz {
	public static void main (String[] args){
		int n;
		int matriz[][] = new int[10][10];
		int soma[] = new int[10];

		n = MyIO.readInt();

		for(int i = 0; i < n; i++){
			for(int j = 0; j < n; j++) {
				matriz[i][j] = MyIO.readInt();
			}
		}
		
		for(int j = 0; j < n; j++){
			soma[j] = 0;
			for(int i = 0; i < n; i++){
				soma[j] += matriz[i][j];
			}
		}

		for(int j = 0; j < n; j++){
			System.out.print(soma[j] + " ");
		}
	}
}

File: teoricos/tt2/processoAEDSIIVerde/Matriz C/matriz.cc
#include <iostream>
using namespace std;

int main(){

	int n;
	int matriz[10][10];
	int soma[10];

	cin >> n;

	for(int i = 0; i < n; i++){
		for(int j = 0; j < n; j++){
			cin >> matriz[i][j];
		}
	}

	for(int j = 0; j < n; j++){
		soma[j] = 0;
		for(int i = 0; i < n; i++){
			soma[j] += matriz[i][j];
		}
	}

	for(int j = 0; j < n; j++){
		cout << soma[j] << " ";
	}
}
